### PR TITLE
Community Tickets Integration and Bug fixes

### PR DESCRIPTION
**Pull request by rafsuntaskin**
Individual API for each event is now shown on the Community Event edit pages:

Screen Shot 2021-06-28 at 4 08 03 PM

### CODE REVIEW

File: src/Tribe/Api_Handler.php [hunk 1]
```diff
@@ -138,4 +138,47 @@ function render_event_tickets_api_meta_box( $post ) {
...
+	function add_community_tickets_api_box() {
+
+		$post = get_post();
+
+		// Only display the API box if we are editing a valid post.
+		if ( ! $post ) {
+			return;
+		}
+
+		$uri = $_SERVER[ 'REQUEST_URI' ];
```
Comment (bordoni) [inline src/Tribe/Api_Handler.php]: Make sure we check if this variable exists before we use it.

File: src/Tribe/Api_Handler.php [hunk 2]
```diff
@@ -138,4 +138,47 @@ function render_event_tickets_api_meta_box( $post ) {
...
+
+		// Maybe generate API key if needed.
+		$this->generate_api_code_for_event( $post->ID );
+		?>
+		<div id="tickets-api" class="tribe-section tribe-section-api-key">
+			<div class="tribe-section-header">
+				<h3>
+					<?php
+					esc_html_e( 'Event API Key', 'et-per-event-checkin' );
+					?>
```
Comment (bordoni) [inline src/Tribe/Api_Handler.php]:
```suggestion
					<?php esc_html_e( 'Event API Key', 'et-per-event-checkin' ); ?>
```

File: src/Tribe/Api_Handler.php [hunk 3]
```diff
@@ -138,4 +138,47 @@ function render_event_tickets_api_meta_box( $post ) {
...
+				</h3>
+			</div>
+			<div class="tribe-section-content">
+				<?php
+					$this->render_event_tickets_api_meta_box( $post );
+				?>
```
Comment (bordoni) [inline src/Tribe/Api_Handler.php]:
```suggestion
				<?php $this->render_event_tickets_api_meta_box( $post ); ?>
```

**Review by bordoni — CHANGES_REQUESTED** [review]

**Review by juanfra — APPROVED** [review]
🚀

**Pull request merged**